src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -236,7 +236,9 @@ function _handleChange(event) {
   var props = this._currentElement.props;
   var returnValue = LinkedValueUtils.executeOnChange(props, event);
 
-  this._wrapperState.pendingUpdate = true;
+  if (this._rootNodeID && !this._wrapperState.pendingUpdate) {
```

Comment:
This seems reasonable to me.  We don't need to check the value of `pendingUpdate`, we can just blindly set it to true.  Can you update the PR to remove that check?